apps/web/src/services/cost-explorer/store/cost-query-set-store.ts

```diff
@@ -7,21 +7,22 @@ import { store } from '@/store';
 import ErrorHandler from '@/common/composables/error/errorHandler';
 
 import { managedCostQuerySets } from '@/services/cost-explorer/cost-analysis/config';
+import type { DataSourceModel } from '@/services/cost-explorer/model';
 import type { CostQuerySetModel } from '@/services/cost-explorer/type';
 
 interface CostAnalysisLNBState {
     costQuerySetList: CostQuerySetModel[];
-    dataSourceList: string[];
+    dataSourceList: DataSourceModel[];
     selectedQuerySetId?: string;
-    selectedDataSource?: string;
+    selectedDataSourceId?: string;
```

Comment:
question: could you add `selectedDataSourceId` to cost-analysis-page-store too?

Reply:
Yes! Should I do it?